tests/js/spec/components/group/sentryAppExternalIssueForm.spec.jsx

```diff
@@ -291,7 +297,10 @@ describe('SentryAppExternalIssueForm Dependent fields', () => {
       expect(boardMock).toHaveBeenCalled();
       expect(wrapper.find('SelectControl#board_id').prop('disabled')).toBe(false);
 
-      wrapper.find('input#board_id').simulate('change', {target: {value: 'b'}});
+      const boardInput = wrapper.find('[data-test-id="board_id"] input').at(0);
+      boardInput.instance().value = 'b';
+      boardInput.simulate('change', {target: {value: 'b'}});
```

Comment:
Is this something we can add to `sentry-test/select-new`

Reply:
@billyvg Not really since I had to add a `data-test-id` to get the right selector.

Reply:
sure but the other logic is repeated, given an input, you have to set its `instance.value` before you can simulate a change. I can see others being tripped up by this

Reply:
@billyvg So pass in an element and then set the value? I don't think this is necessarily unique to react select but if you think that's the right place to put it, I can do that.

Reply:
ok so maybe backing up then, why are you setting the component's instance value?

Reply:
@billyvg It's to simulate a user typing into the text field which will then call the endpoint based off the specified `uri` (the async behavior). I ended up putting the utility function in that file but it could be used for just about any kind of text input.